### PR TITLE
Updated markup in DRAGONS notebooks

### PR DESCRIPTION
**Pull request by bmerino95**
I have incorporated David's suggestions into the notebooks and updated __version__ for each notebook. All notebooks run on GP13.

### CODE REVIEW

**Comment by rnikutta** [comment]
@bmerino95 thanks. Can you request @kareninysimba to review these latest changes?

**Comment by bmerino95** [comment]
@rnikutta Good idea! I just added him as a reviewer.

**Comment by rnikutta** [comment]
@kareninysimba any chance your could greenlight this PR today? (I'd like to merge it first before merging @bmerino95's Acks&Disclaimer PR. Thanks!

File: 04_HowTos/DataReduction/DRAGONS_reduction_examples/GSAOI_Imaging_EllipticalGalaxy/GSAOI_Imaging_EllipticalGalaxy.ipynb [hunk 1]
```diff
@@ -295,7 +295,9 @@
     "<a class=\"anchor\" id=\"Master_Flat\"></a>\n",
     "# Create master flat field\n",
     "\n",
-    "The GSAOI Kshort master flat is created from a series of lamp-on and lamp-off dome exposures. They should all have the same exposure time. Each flavor is stacked (averaged), then the lamp-off stack is subtracted from the lamp-on stack, and the result is normalized."
+    "The GSAOI Kshort master flat is created from a series of lamp-on and lamp-off dome exposures. They should all have the same exposure time. Each flavor is stacked (averaged), then the lamp-off stack is subtracted from the lamp-on stack, and the result is normalized.\n",
+    "\n",
+    "The master flat will be saved as S20170505S0030_flat.fits."
```
Comment (kareninysimba) [inline 04_HowTos/DataReduction/DRAGONS_reduction_examples/GSAOI_Imaging_EllipticalGalaxy/GSAOI_Imaging_EllipticalGalaxy.ipynb:300]: Change
_"The master flat will be saved as S20170505S0030\_flat.fits."_
with
_"The master flat will be saved with the \_flat.fits."_

File: 04_HowTos/DataReduction/DRAGONS_reduction_examples/GSAOI_Imaging_EllipticalGalaxy/GSAOI_Imaging_EllipticalGalaxy.ipynb [hunk 2]
```diff
@@ -350,7 +354,7 @@
     "\n",
     "The sky frame comes from off-target sky observations. We feed the pipeline all the on-target and off-target frames. The software will appropriately split the on-target and off-target using information in the headers.\n",
     "\n",
-    "This command will generate flat-corrected and sky-subtracted files but will not stack them. You can differentiate each file by its suffix (_flatCorrected or _skyCorrected). The on-target files are the ones that have been sky subtracted (_skyCorrected). There should be nine of them.\n",
+    "This command will generate flat-corrected and sky-subtracted files but will not stack them. You can differentiate each file by its suffix (_flatCorrected or _skyCorrected). The on-target files are the ones that have been sky subtracted (_skyCorrected). Finally, the stacked image will be saved as S20170505S0095_image.fits.\n",
```
Comment (kareninysimba) [inline 04_HowTos/DataReduction/DRAGONS_reduction_examples/GSAOI_Imaging_EllipticalGalaxy/GSAOI_Imaging_EllipticalGalaxy.ipynb:357]: Assuming there won't be other images before with the suffix _image.fits, change
_"the stacked image will be saved as S20170505S0095\_image.fits."_
with
_"the stacked image will be saved with the suffix \_image.fits."_

File: 04_HowTos/DataReduction/DRAGONS_reduction_examples/NIRI_Imaging_Supernova/NIRI_Imaging_Supernova.ipynb [hunk 3]
```diff
@@ -398,7 +398,9 @@
     "<a class=\"anchor\" id=\"Master_Flat\"></a>\n",
     "# Master flat field\n",
     "\n",
-    "A NIRI master flat is created from a series of lamp-on and lamp-off exposures. Each flavor is stacked, and then the lamp-off stack is subtracted from the lamp-on stack."
+    "A NIRI master flat is created from a series of lamp-on and lamp-off exposures. Each flavor is stacked, and then the lamp-off stack is subtracted from the lamp-on stack.\n",
+    "\n",
+    "The master flat will be saved as N20160102S0373_flat.fits."
```
Comment (kareninysimba) [inline 04_HowTos/DataReduction/DRAGONS_reduction_examples/NIRI_Imaging_Supernova/NIRI_Imaging_Supernova.ipynb:403]: Change
_"The master flat will be saved as N20160102S0373\_flat.fits."_
with
_"The master flat will be saved with the suffix \_flats.fits"_

File: 04_HowTos/DataReduction/DRAGONS_reduction_examples/NIRI_Imaging_Supernova/NIRI_Imaging_Supernova.ipynb [hunk 4]
```diff
@@ -453,7 +457,9 @@
     "\n",
     "The master dark and flat will be retrieved automatically from the local calibration database. Again, the user BPM needs to be specified on the command line.\n",
     "\n",
-    "The output stack units are in electrons (header keyword BUNIT=electrons). The output stack is stored in a multi-extension FITS (MEF) file. The science signal is in the \"SCI\" extension, the variance is in the \"VAR\" extension, and the data quality plane (mask) is in the \"DQ\" extension."
+    "The output stack units are in electrons (header keyword BUNIT=electrons). The output stack is stored in a multi-extension FITS (MEF) file. The science signal is in the \"SCI\" extension, the variance is in the \"VAR\" extension, and the data quality plane (mask) is in the \"DQ\" extension.\n",
+    "\n",
+    "The final reduced image will be saved as N20160102S0296_image.fits."
```
Comment (kareninysimba) [inline 04_HowTos/DataReduction/DRAGONS_reduction_examples/NIRI_Imaging_Supernova/NIRI_Imaging_Supernova.ipynb:462]: Change
_"The final reduced image will be saved as N20160102S0296\_image.fits."_
with
_"The final reduced image will be saved with the suffix \_image.fits."_

**Review by kareninysimba — COMMENTED** [review]
@bmerino95 The general scope of these suggestion, as you'll see, is to make the notebooks more generic for when users apply them to their own data.
That's why I suggest to try to eliminate stating the actual, full name of the file but only specifying if it's the first of a list, or just that it has the _xxx.fits suffix.

Otherwise, everything is much clearer and better explained!

**Comment by kareninysimba** [comment]
@rnikutta I made suggestions about the original scope of my initial suggestions. For the sake of hurry, @bmerino95 can skip them.
Only real typo is in comment #2 (for [GMOS_Imaging_Galaxy.ipynb](https://github.com/astro-datalab/notebooks-latest/pull/232/files/ad99c89f05092b89ef82b43bbe8df0b8263efd4a#diff-1c185aec81b4446ce67e6721173989cb8a5b04d5bdb3ab707ad77e1190f91d7c), where I think it should say _fringe_ instead of _flat_.

**Comment by bmerino95** [comment]
Hi @kareninysimba. Thank you for your feedback. I have made the changes you recommended and made the statement for the standard star in the NIRI notebook more generic as well.

File: 04_HowTos/DataReduction/DRAGONS_reduction_examples/GMOS_longslit_WhiteDwarf/GMOS_longslit_WhiteDwarf.ipynb [hunk 5]
```diff
@@ -432,7 +432,9 @@
     "\n",
     "We create the master biases with the Reduce class. We will run it twice, once for each of the two raw bias lists. The master biases will be automatically added to the local calibration manager when the \"store\" parameter is present in the .dragonsrc configuration file. The output is written to disk and its name is stored in the Reduce instance. The calibration service expects the name of a file on disk.\n",
     "\n",
-    "Because the database was given the \"store\" option in the dragonsrc file, the processed biases will be automatically added to the database at the end of the recipe."
+    "Because the database was given the \"store\" option in the dragonsrc file, the processed biases will be automatically added to the database at the end of the recipe. \n",
+    "\n",
+    "When the cell is done running, the master biases will be saved as S20170825S0347_bias.fits and S20171021S0265_bias.fits."
```
[the remark this answers is not in the document]
Comment (kareninysimba) [reply 04_HowTos/DataReduction/DRAGONS_reduction_examples/GMOS_longslit_WhiteDwarf/GMOS_longslit_WhiteDwarf.ipynb:437]: This one is pending still

**Review by kareninysimba — COMMENTED** [review]
Excellent job! There's only 1 suggestion missing (I commented on it)

**Comment by bmerino95** [comment]
Thank you for catching that. I just changed the text under 'Create master bias' in GMOS_Longslit_WhiteDwarf.ipynb.

File: 04_HowTos/DataReduction/DRAGONS_reduction_examples/GMOS_longslit_WhiteDwarf/GMOS_longslit_WhiteDwarf.ipynb [hunk 6]
```diff
@@ -434,7 +434,9 @@
     "\n",
     "Because the database was given the \"store\" option in the dragonsrc file, the processed biases will be automatically added to the database at the end of the recipe. \n",
     "\n",
-    "When the cell is done running, the master biases will be saved as S20170825S0347_bias.fits and S20171021S0265_bias.fits."
+    "When the cell is done running, the master biases will be saved as S20170825S0347_bias.fits and S20171021S0265_bias.fits.\n",
```
Comment (kareninysimba) [inline 04_HowTos/DataReduction/DRAGONS_reduction_examples/GMOS_longslit_WhiteDwarf/GMOS_longslit_WhiteDwarf.ipynb:437]: New comment added! but old one still present. Almost there!

**Review by kareninysimba — COMMENTED** [review]
1 to go!

**Comment by bmerino95** [comment]
Saw the old text there and removed it. Now, just the updated text is there.

**Review by kareninysimba — APPROVED** [review]
Yay! Congrats. Changes I suggested seem implemented successfully.

**Pull request merged**